push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/GCMPushNotificationSender.java

```diff
@@ -80,6 +81,10 @@ public void sendPushMessage(Variant variant, Collection<String> tokens, UnifiedP
         gcmBuilder.addData("alert", message.getAlert());
         gcmBuilder.addData("sound", message.getSound());
         gcmBuilder.addData("badge", "" + message.getBadge());
+        gcmBuilder.priority(message.getPriority() ==     Priority.HIGH ? 
```

Comment:
Let's add a comment that 'normal' is Googles default (not our), and that abusage of HIGH might cause problems for the app. I think it's good info to have hear, as changing to HIGH (for whatever reasons) might be tempting :)